README.md

```diff
@@ -400,18 +400,18 @@ Other Style Guides
     // good
     [1, 2, 3].map(x => x + 1);
 
-    // bad
+    // very bad - error bad - no returned value means `memo` becomes undefined after the first iteration
```

Comment:
i think just "bad" here, with your english explanation, is sufficient

Reply:
That's fair. I've taken it back down just to "bad"